integration-tests/tests/billing-projects.js

```diff
@@ -38,16 +38,12 @@ const billingProjectsPage = (testPage, testUrl) => {
       return click(testPage, clickable({ text }))
     },
 
-    selectProjectMenu: async () => {
-      await click(testPage, clickable({ text: 'Billing project menu' }))
-    },
-
-    selectDeleteProjectMenuOption: async () => {
-      await click(testPage, clickable({ text: 'Delete Billing Project' }))
+    selectDeleteProjectMenuOption: async projectName => {
+      await click(testPage, clickable({ text: `Delete billing project ${projectName}` }))
     },
 
     confirmDeleteBillingProject: async () => {
-      await click(testPage, clickable({ textContains: 'Delete' }))
+      await click(testPage, clickable({ text: 'Delete billing project' }))
```

Comment:
Has to be exact now to avoid clicking on the wrong thing (trash can button instead of confirmation button).